### DISCUSSION TITLE
Install on Digital Ocean Droplet or App?

### DISCUSSION THREAD

**Discussion by osseonews** [PostgREST/postgrest#3053] [Q&A] [answered]
Is there any tutorial for how to set this up on Digital Ocean droplet or App, so you can immediately have a restful API on a managed do database? I saw the video for setting this up locally with a DO database and it's great. But, it didn't explain how to actually set this up on a DO app or droplet. thanks.

**Comment by laurenceisla** [comment]
I think [the Part 3 of this tutorial](https://www.youtube.com/watch?v=TfvmfrGD5EQ) mentions how to deploy PostgREST to a Digital Ocean droplet. Check Part 1 and 2 if you want to get the full context.

**Comment by osseonews** [comment]
Thanks. I watched that and tried to install it, but it doesn't work. I've installed other applications with docker and they work fine. can't for the life of me figure out how to get this running. Below is my docker run command. i need to this to run on port 8080, not 3000. Any ideas? $URI is just a placeholder for our real URI.

`docker run -p 8080:80 -e PGRST_DB_URI="$URI" -e PGRST_DB_ANON_ROLE="anon" -e PGRST_SERVER_PORT="8080" -d postgrest/postgrest`

**Reply by wolfgangwalther** [reply, marked as answer]
> ``` 
> docker run -p 8080:80
> ``` 

This will expose the port 80 inside the container to port 8080 outside.

> ``` 
> -e PGRST_SERVER_PORT="8080"
> ``` 

This will make postgrest listen on port 8080 inside the container.

Your ports don't match up.

Do this instead:

```
docker run -p 8080:3000 -e PGRST_DB_URI="$URI" -e PGRST_DB_ANON_ROLE="anon" -d postgrest/postgrest
```

**Reply by osseonews** [reply]
Simple. Thank you so much! That worked.
